packages/arb-token-bridge-ui/src/components/TransferPanel/WithdrawalConfirmationDialog.tsx

```diff
@@ -88,7 +56,9 @@ function FastBridgesTable() {
     trackEvent(`Fast Bridge Click: ${bridgeName}`)
   }
 
-  const sortedFastBridges = [...FastBridges].sort((a, b) => {
+  const sortedFastBridges = [
+    ...getFastBridges(from.chainID, to.chainID, selectedToken?.symbol || 'ETH')
```

Comment:
for withdrawal we have displayed the full list before regardless of a token, so that we let the bridge itself decide what to do with a token they don't support: some of them default to eth, some leave selected token field empty.

with deposits it's not possible to select a token that's not supported as we specify the supported list ourselves.